Assets/VRTK/Scripts/Utilities/ObjectFollow/VRTK_RigidbodyFollow.cs

```diff
@@ -99,5 +101,16 @@ protected override Vector3 GetScaleToFollow()
         {
             return rigidbodyToFollow.transform.localScale;
         }
+
+        private void CacheRigidbodies()
```

Comment:
Everything should now be at least `protected virtual`

I should probably update the contrib guidelines with that!